winui/DataGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,654 @@
+
+
+Here, `OrderID` column is displayed in `DetailsViewDataGrid` and it is excluded while exporting to excel.
+
+### Customizing DetailsViewDataGrid cells
+
+Like parent DataGrid, You can customize the `DetailsViewDataGrid` cells also by using `CellsExportHandler`. Based on `DataGridCellExcelExportOptions.GridViewDefinition` property, you can identify the particular `DetailsViewDataGrid` and customize it.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.CellsExportHandler = CellsExportHandler;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private static void CellsExportHandler(object sender, DataGridCellExcelExportOptions e)
+{
+    if (e.GridViewDefinition == null)
+        return;
+
+    if (e.ColumnName == "OrderID")
+    {
+        e.Range.CellStyle.Font.Size = 12;
+        e.Range.CellStyle.Font.Color = ExcelKnownColors.Blue;
+        e.Range.CellStyle.Font.FontName = "Segoe UI";
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-exported-cell-customization.png" alt="Customizing DetailsViewDataGrid Cells while Exporting to Excel in WinUI DataGrid" width="100%" Height="Auto"/>
+
+
+## Performance
+
+Using `DataGridExcelExportOptions.CellsExportHandler` and changing settings for each cell will consume more memory and time consumption. So, avoid using `CellsExportHandler` and instead of you can do the required settings in the exported sheet.
+ 
+### Formatting column without using CellsExportHandler
+
+You can perform cell level customization such as row-level styling, formatting particular column in the exported worksheet. 
+
+In the below code snippet, NumberFormat for `Unit Price` column is changed in the exported sheet after exporting without using `CellsExportHandler`. 
+
+Reference:
+[http://help.syncfusion.com/file-formats/xlsio/working-with-cell-or-range-formatting](http://help.syncfusion.com/file-formats/xlsio/working-with-cell-or-range-formatting)
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+IWorkbook workBook = excelEngine.Excel.Workbooks[0];
+workBook.ActiveSheet.Columns[1].NumberFormat = "0.0";
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-column-formatting.png" alt="Formatting the Column while Exporting to Excel in WinUI DataGrid" width="100%" Height="Auto"/>
+
+### Alternate row styling without using CellsExportHandler
+
+In the below code snippet, the background color of rows in excel is changed based on row index using conditional formatting for better performance.
+
+Reference:
```

Comment:
Can we change the Text in the reference?

Reply:
modified